# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Jakub Kadlčík <frostyx@email.cz>, 2015
msgid ""
msgstr ""
"Project-Id-Version: tracer\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-04-05 14:32+0200\n"
"PO-Revision-Date: 2015-04-05 12:35+0000\n"
"Last-Translator: Jakub Kadlčík <frostyx@email.cz>\n"
"Language-Team: Czech (http://www.transifex.com/projects/p/tracer/language/cs/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: cs\n"
"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"

#: ../tracer/controllers/default.py:84
msgid "Press application number for help or 'q' to quit"
msgstr "Pro zobrazení nápovědy zadejte číslo aplikace. Nebo 'q' pro ukončení"

#: ../tracer/controllers/default.py:92
msgid "Wrong application number"
msgstr "Neplatné číslo aplikace"

#: ../tracer/controllers/default.py:94
msgid "Press <enter> to get list of applications"
msgstr "Pro vypsání seznamu aplikací stisknete <enter>"

#: ../tracer/controllers/helper.py:55
msgid "You don't have enough permissions"
msgstr "Nemáte dostatečné oprávnění"

#: ../tracer/controllers/helper.py:65
#, python-brace-format
msgid "Application called {0} is not running"
msgstr "Aplikace s názvem {0} není v tuto chvíli spuštěna"

#: ../tracer/resources/applications.py:112
msgid "You will have to reboot your computer"
msgstr "Bude potřeba restartovat počítač"

#: ../tracer/resources/applications.py:115
msgid "You will have to log out & log in again"
msgstr "Bude potřeba odhlášení a opětovné přihlášení do systému"

#: ../tracer/resources/router.py:46
msgid "Only root can use this application"
msgstr "Tato aplikace vyžaduje oprávnění uživatele root"

#: ../tracer/resources/exceptions.py:30
#, python-brace-format
msgid ""
"You are running unsupported linux distribution\n"
"\n"
"Please visit https://github.com/FrostyX/tracer/issues\n"
"and create new issue called 'Unknown or unsupported linux distribution: {0} (v{1})' if there isn't such.\n"
"\n"
"Don't you have an GitHub account? Please report this issue on frostyx@email.cz"
msgstr "Používáte nepodporovanou linuxovou distribuci.\n\nProsím navštivte https://github.com/FrostyX/tracer/issues\na vytvořte nový problém s názvem 'Unknown or unsupported linux distribution: {0} (v{1})', pokud již neexistuje.\n\nNemáte uživatelský účet na portálu GitHub? Prosím nahlaste problém přes frostyx@email.cz"

#: ../tracer/resources/exceptions.py:45
msgid "Package database is locked by another process"
msgstr "Databáze balíčků je uzamčena jiným procesem"

#: ../tracer/resources/exceptions.py:55
msgid "You can't open package database due to insufficient permissions"
msgstr "Pro otevření databáze balíčků nemáte dostatečné oprávnění "

#: ../tracer/resources/exceptions.py:66
#, python-brace-format
msgid ""
"Problem occurred - neither one of {0} paths exists\n"
"Please contact maintainer of tracer package in your distribution."
msgstr "Vyskytl se problém - ani jedno z {0} umístění neexistuje.\nProsím kontaktuje správce balíčku tracer pro vaši distribuci."

#: ../tracer/views/interactive.py:11 ../tracer/views/default.py:70
msgid "You should restart:"
msgstr "Bylo by vhodné restartovat:"

#: ../tracer/views/default.py:51
msgid "Some applications using:"
msgstr "Některé aplikace pomocí:"

#: ../tracer/views/default.py:52
msgid "These applications manually:"
msgstr "Následující aplikace ručně:"

#: ../tracer/views/default.py:57
msgid "These applications restarting your session:"
msgstr "Následující aplikace restartováním vašeho sezení:"

#: ../tracer/views/default.py:61
msgid "These applications rebooting your computer:"
msgstr "Následující aplikace restartováním počítače:"

#: ../tracer/views/helper.py:46
msgid "How to restart"
msgstr "Jak restartovat"

#: ../tracer/views/helper.py:57
msgid "Affected by"
msgstr "Ovlivněno kvůi"

#: ../tracer/views/note_for_hidden.py:13
msgid "There are:"
msgstr "Nalezeno:"

#: ../tracer/views/note_for_hidden.py:15
msgid "Additionally to those process above, there are:"
msgstr "Kromě aplikací vypsaných výše, je zde:"

#: ../tracer/views/note_for_hidden.py:19
#, python-brace-format
msgid ""
"{0} processes requiring restart of your session (i.e. Logging out & Logging "
"in again)"
msgstr "{0} procesů vyžadujících restart vašeho sezení (i.e. odhlášení a opětovné přihlášení)"

#: ../tracer/views/note_for_hidden.py:25
#, python-brace-format
msgid "{0} processes requiring reboot"
msgstr "{0} procesů vyžadujících reboot"
